app/views/subscriptions/new.html.erb

```diff
@@ -5,10 +5,19 @@
     <%= form_tag(subscriptions_path, method: :post, class: 'subscription-form') do %>
       <fieldset>
         <div class="row">
-          <%= label_tag('subscription[description]', 'Description', class: 'eui-required-o') %>
+          <%= label_tag('subscription[name]', 'Subscription Name', class: 'eui-required-o') %>
         </div>
         <div class="row">
-          <%= text_area_tag('subscription[description]', @subscription['description'], cols: 30, rows: 3, class: 'required textcounter', maxlength: 255) %>
+          <%= text_area_tag('subscription[name]', @subscription['name'], cols: 30, rows: 3, class: 'required textcounter', maxlength: 255) %>
+        </div>
+      </fieldset>
+      
+      <fieldset>
+        <div class="row">
+          <%= label_tag('subscription[concept_id]', 'Collection Concept ID', class: 'eui-required-o') %>
+        </div>
+        <div class="row">
+          <%= text_field_tag('subscription[concept_id]', @subscription['concept_id'], class: 'required full-width', placeholder: 'Enter a ConceptID for a single collection. (Example: C1234567-PODAAC)') %>
```

Comment:
I'm not a fan of the placeholder text (UI/UX team doesn't like it for design reasons), but I asked PO about it and we can make tweaks as we go along. So we can leave it, just wanted to register my thoughts.